### PR TITLE
DIS **-011: remove unused definitions

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes cplusplus/nbballot#553
Fixes cplusplus/nbballot#558

### CODE REVIEW

File: source/intro.tex [hunk 1]
```diff
@@ -359,20 +349,6 @@
 \definition{implementation limits}{defns.impl.limits}
 restrictions imposed upon programs by the implementation
 
-\definition{iostream class templates}{defns.iostream.templates}
```
Comment (jwakely) [inline source/intro.tex:362]: This is referred to in a note in [defns.character.container]:

"It is used for one of the template parameters of the string, iostream, and regular expression class
templates."

Comment (frederick-vs-ja) [reply source/intro.tex:362]: #6236 will also be fixed once [defns.iostream.templates] is correctly removed by this PR.

Comment (jwakely) [reply source/intro.tex:362]: Ah yes, I knew I'd discussed this defn recently. I still think my suggestion there would improve [defns.character.container].

Comment (tkoeppe) [reply source/intro.tex:362]: We'll also get to your suggestion!

**Pull request merged**